stackernews/stacker.news
# Add PWA feature to stacker news

Summary:

I am adding PWA feature to stacker news so that It can be installed on a phone/desktop like a native app.

Testing Instructions:

1. Once you have switched to this branch then open the site in a browser of a phone or on the desktop computer and you should get an option to install it as a PWA like this ![](https://d.pr/i/m62xik+)
2. If any issues, feel free to leave a comment.
